umm-spec-lib/src/cmr/umm_spec/migration/version_migration.clj

```diff
@@ -17,13 +19,30 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;; Utility Functions
 
+(defn- customized-compare
+  "Compare begin and end.
```

Comment:
"1.1" vs "1.10" is fine. it's other ones compared to "1.10". it does the string compare and they are all bigger than "1.10".  Done.